src/Private/Helpers.ps1

```diff
@@ -3059,3 +3075,235 @@ function Test-PodeVersionPwshEOL {
+            }
+            # prevent these values being identified as an object
+            if ($InputObject -is [System.Collections.Specialized.OrderedDictionary]) {
+                $Type = 'HashTable'
+            }
+            elseif ($Type -ieq 'List`1') {
+                $Type = 'Array'
+            }
+            elseif ($InputObject -is [array]) {
+                $Type = 'Array'
+            } # whatever it thinks it is called
+            elseif ($InputObject -is [hashtable]) {
+                $Type = 'HashTable'
+            } # for our purposes it is a hashtable
+
+            $output += switch ($Type.ToLower()) {
+                'string' {
+                    $String = "$InputObject"
+                    if (($string -match '[\r\n]' -or $string.Length -gt 80) -and ($string -notlike 'http*')) {
+                        $folded = [System.Text.StringBuilder]::new(">`n") # signal that we are going to use the readable 'newlines-folded' format
+                        foreach ($item in $string.Split("`n")) {
+                            $workingString = $item -replace '\r$'
+                            $length = $item.Length
+                            $IndexIntoString = 0
+                            $wrap = 80
+                            while ($length -gt $IndexIntoString + $Wrap) {
+                                $BreakPoint = $wrap
+                                $earliest = $workingString.Substring($IndexIntoString, $wrap).LastIndexOf(' ')
+                                $latest = $workingString.Substring($IndexIntoString + $wrap).IndexOf(' ')
+                                if (($earliest -eq -1) -or ($latest -eq -1)) {
+                                    $BreakPoint = $wrap
+                                }
+                                elseif ($wrap - $earliest -lt ($latest)) {
+                                    $BreakPoint = $earliest
+                                }
+                                else {
+                                    $BreakPoint = $wrap + $latest
+                                }
+                                if (($wrap - $earliest) + $latest -gt 30) {
+                                    $BreakPoint = $wrap # in case it is a string without spaces
+                                }
+
+                                $null = $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString, $BreakPoint).Trim())
+                                $IndexIntoString += $BreakPoint
+                            }
+                            if ($IndexIntoString -lt $length) {
+                                $null = $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString).Trim())
+                            }
+                            else {
+                                $null = $folded.AppendLine()
+                            }
+                        }
+                        $folded.ToString()
+                        break
```

Comment:
I've had a play with this multiline string formatter after spotting some oddities from the PetStore example - such as lines longer than 80 characters, or spaces in the output where there shouldn't be any between characters in a word.

For lines 3216-3250, I've tweaked then to be the following. This now keeps lines to <=80 chars, and also puts line-break escapes in for lines which have to be split of multiple lines with no spaces - such as URLs. You'll also spot I changed the string start from `>` to `|`, as I feel this looks better with newlines preserved in the editor and swagger/etc, so they can render bullet points:

This now:
![image](https://github.com/Badgerati/Pode/assets/1483395/976a597b-4431-402b-b318-0c3a4c1195e3)

Instead of:
![image](https://github.com/Badgerati/Pode/assets/1483395/87b88829-30fd-40d1-a0f4-42b0d9233b01)

Here's the code:
```powershell
$multiline = [System.Text.StringBuilder]::new("|`n")

foreach ($item in $string.Split("`n")) {
    $workingString = $item -replace '\r$'
    $length = $workingString.Length
    $index = 0
    $wrap = 80
    while ($index -lt $length) {
        $breakpoint = $wrap
        $linebreak = $false

        if (($length - $index) -gt $wrap) {
            $lastSpaceIndex = $workingString.LastIndexOf(' ', $index + $wrap, $wrap)
            if ($lastSpaceIndex -ne -1) {
                $breakpoint = $lastSpaceIndex - $index
            }
            else {
                $linebreak = $true
                $breakpoint--
            }
        }
        else {
            $breakpoint = $length - $index
        }

        $null = $multiline.Append($padding).Append($workingString.Substring($index, $breakpoint).Trim())
        if ($linebreak) {
            $null = $multiline.Append('\')
        }
        $null = $multiline.Append([System.Environment]::NewLine)
        $index += $breakpoint
    }
}
$multiline.ToString().TrimEnd()
break
```

Reply:
Just tweaked the code above slightly, as I spotted a minor newline issue.

Reply:
Nice improvement :)